ursabot/factories.py

```diff
@@ -79,8 +79,12 @@ def prepend_step(self, step):
                repourl='https://github.com/ursa-labs/ursabot',
                mode='full'),
     ShellCommand(command=['ls', '-lah']),
-    ShellCommand(command=['pip', 'install', 'pytest', 'flake8']),
-    ShellCommand(command=['pip', 'install', '-e', '.']),
+    ShellCommand(command=['pip', 'install', 'pytest', 'flake8', 'mock']),
```

Comment:
Perhaps some requirements file is useful at this point?

Reply:
Created issue: https://github.com/ursa-labs/ursabot/issues/28